code/modules/reqs/supplypacks.dm

```diff
@@ -424,7 +424,7 @@ AMMO
 		/obj/item/cell/lasgun/tesla,
 		/obj/item/cell/lasgun/tesla,
 	)
-	cost = 30
+	cost = 15
```

Comment:
```suggestion
	cost = 10
```
just make it ten, the gun isn't very good already, or you should just make it 3 points and only be 1 item like all other power gun things

Reply:
ok